src/iperf_server_api.c

```diff
@@ -562,7 +545,6 @@ iperf_run_server(struct iperf_test *test)
 	}
 
         memcpy(&read_set, &test->read_set, sizeof(fd_set));
-        memcpy(&write_set, &test->write_set, sizeof(fd_set));
```

Comment:
Removing this means that `write_set` may have an uninitialized values on line 581. Should you just remove the variable entirely since it it unused?

Reply:
Thanks for the comment.  Forgot to remove all occurances of `write_set`...  Now removed (with rebase).